src/core/recv_buffer.h

```diff
@@ -22,9 +25,19 @@ typedef struct QUIC_RECV_CHUNK {
     CXPLAT_LIST_ENTRY Link;         // Link in the list of chunks.
     uint32_t AllocLength : 31;      // Allocation size of Buffer
     uint32_t ExternalReference : 1; // Indicates the buffer is being used externally.
-    uint8_t Buffer[0];
+    uint8_t *Buffer;                // Pointer to the buffer itself. Doesn't need to be freed independently:
```

Comment:
Somewhat independent of this PR, I am wondering if we could/should annotate this pointer:
```suggestion
    _Field_size_bytes_(AllocLength)
    uint8_t *Buffer;                // Pointer to the buffer itself. Doesn't need to be freed independently:
```